internal/component/loki/process/stages/labels_test.go

```diff
@@ -105,18 +103,18 @@ func TestLabels(t *testing.T) {
 				"l3": &lv3,
 			}},
 			err: nil,
-			expectedCfgs: LabelsConfig{Values: map[string]*string{
-				"l1": &lv1,
-				"l2": &lv2c,
-				"l3": &lv3c,
-			}},
+			expectedCfgs: map[string]string{
+				"l1": "lv1",
+				"l2": "l2",
+				"l3": "l3",
```

Comment:
Since the map's value is no longer a pointer to string, why not just use `"l1": lv1`?

Reply:
ok